content/community/nominating-issues/how-to-nominating-issues.adoc

```diff
@@ -0,0 +1,38 @@
+---
+title: "Nominate Issues"
+weight: 10
+---
+
+:imagesdir: /images/en/community/
+
+== How to Nominate Issues
+'''
+
+The following instructions are to help guide you through how to nominate your/community issues that we will tackle in weekly sprints.
+
+
+1 - Leave a comment following the template we have provided. The format must be followed below. Please Provide the link to the issue and justify why you would like that issue in the our sprint.
```

Comment:
Use the normal automatic Asciidoc numbering, instead of manual

Reply:
The Asciidoc numbering such as ```[start=1]``` with dots? I'm not sure that will work here due to the code block tags below.

Reply:
`start` tag should not be necessary

You can control how asciidoc connects numbered paragraphs by ending lines with `+`, so it knows you're still in the same paragraph even though there is a block there. [Example](https://github.com/salesagility/SuiteDocs/edit/master/content/admin/installation-guide/Downloading%20&%20Installing.adoc)

But if you can't get it to work, or it proves too cumbersome to do, just use the manual numbering but keep the existing format: `1. ` instead of `1- `